px-context-browser.html

```diff
@@ -167,8 +168,8 @@
           </template>
           <div slot="columns" class="context-browser-favorites--empty__container">
             <px-icon icon="px-nav:favorite" class="context-browser-favorites--empty__icon"></px-icon>
-            <div class="context-browser-favorites--empty__text regular">You don't have any favorites yet.</div>
-            <div class="context-browser-favorites--empty__subtext zeta">To favorite a context, click on the star while viewing a context in the context browser.</div>
+            <div class="context-browser-favorites--empty__text regular">[[localize("You don't have any favorites yet.")]]</div>
```

Comment:
@davidrleonard should these be in single quotes too?

Reply:
Nah that’s fine it’s in HTML so it doesn’t matter that much